tests/Language/LanguageServiceTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @copyright Copyright (c) Wizacha
+ * @license Proprietary
+ */
+declare(strict_types = 1);
+
+namespace Wizaplace\SDK\Tests\Language;
+
+use Wizaplace\SDK\Cms\CmsService;
+use Wizaplace\SDK\Language\LanguageService;
+use Wizaplace\SDK\Tests\ApiTestCase;
```

Comment:
regrouper les uses sous Wizaplace\SDK
```suggestion
use Wizaplace\SDK\{
    Cms\CmsService,
    Language\LanguageService,
    Tests\ApiTestCase
};
```